redirects_generator/Properties/launchSettings.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Out of interest: What's this?

Reply:
This is a tool I wrote to make it easier to define a version link between a .NET Framework article and .NET 6+